content/concepts/tokens/nftoken-auctions.md

```diff
@@ -0,0 +1,64 @@
+---
+html: nftoken-auctions.html
+parent: non-fungible-tokens.html
+blurb: You can assign another account to mint NFTs in your stead.
+labels:
+ - Non-fungible Tokens, NFTs
+---
+
+# Running an NFT Auction
+
+There are several ways to run an auction, each with advantages and disadvantages.
+
+## Run the Auction Off the XRPL, Complete the Purchase on XRPL
+
+This flow is the most straightforward. Note that the `NFTokenOffer` objects can always be canceled by their creator, so it's not possible to implement a binding offer.
+
+1. Store your bids in a private database.
```

Comment:
just curious if second person phrasing is intentional here? The assumption is that "you" is the broker? I think the rest of the documentation uses third person

Reply:
You're correct. Typically, we write in second person. I made the mistake because I took the text from the flowchart. On reflection, I think the second person works well in the first scenario, third person in the second and third scenarios. I'm going to leave it for now.